tests/e2e_test.go

```diff
@@ -110,6 +110,7 @@ func TestUploadVerifyRpm(t *testing.T) {
 }
 
 func TestLogInfo(t *testing.T) {
+	t.Skip("TODO (priyawadhwa@): skipping for now, turn on once rekor signs the log root itself")
```

Comment:
cc @dlorenc @lukehinds @bobcallaway -- this test fails when log-info runs `VerifySignedLogRoot` -- I'm guessing it's because the public key we return now is not the one in trillian.

I added a skip in with plans to turn the test on once we start signing roots ourselves, but if anyone has other ideas lmk!

Reply:
I hit this earlier but after a rebase all was well.

Reply:
still failing locally 😞

Reply:
Added in signing of the signature for the log root, this should be working now!